From 6c33a0fd20a2e5c9c1373af116c25b4398c99e59 Mon Sep 17 00:00:00 2001
From: Campbell Jones <git@serebit.com>
Date: Sun, 25 Sep 2022 15:14:13 -0400
Subject: [PATCH] Remove numeric space in formatted clock applet output

This was introduced in the glib2 pull request !2698 and is not desired behavior for us.
---
 src/applets/clock/ClockApplet.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/applets/clock/ClockApplet.vala b/src/applets/clock/ClockApplet.vala
index 85d4fe04..02bdecf0 100644
--- a/src/applets/clock/ClockApplet.vala
+++ b/src/applets/clock/ClockApplet.vala
@@ -323,7 +323,7 @@ public class ClockApplet : Budgie.Applet {
 			ftime = "<small>%s</small>";
 		}
 
-		var formatted = ftime.printf(ctime);
+		var formatted = ftime.printf(ctime).replace("\u2007", "");
 		if (old == formatted) {
 			return true;
 		}
